packages/office-ui-fabric-react/src/components/Nav/Nav.Props.ts

```diff
@@ -173,6 +183,12 @@ export interface INavLink {
   parentId?: string;
 
   /**
+   * (Optional) By default, any link with onClick defined will render as a button.
+   * Set this property to true to override that behavior.
+   */
+  isAnchor?: boolean;
```

Comment:
If a link was previously an anchor (no onclick), it will still be an anchor, no need to set this flag. This is only for my new special case: links that have a click handler *and* should render as anchors.